server/segment_rpc.go

```diff
@@ -381,7 +381,7 @@ func SubmitSegment(sess *BroadcastSession, seg *stream.HLSSegment, nonce uint64)
 		}
 		return nil, fmt.Errorf(errorString)
 	}
-	glog.Infof("Uploaded segment nonce=%d seqNo=%d orch=%s", nonce, seg.SeqNo, ti.Transcoder)
+	glog.Infof("Uploaded segment nonce=%d seqNo=%d orch=%s updur=%s", nonce, seg.SeqNo, ti.Transcoder, uploadDur)
```

Comment:
Same note about `dur` . Also, do we happen to have the manifestID accessible that we can put here? `nonce=... manifestID=... seqNo=...`

(I actually was going to submit a PR which adds exactly this but will hold off if it can just be added here)

Reply:
added